lib/myfinance/resources/entity.rb

```diff
@@ -31,7 +31,7 @@ def find_all
       #
       def find(entity_id)
         http.get("/entities/#{entity_id}", body: {}) do |response|
-          respond_with_object(response)
+          respond_with_object(response, 'entity')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.